src/averageAges.js

```diff
@@ -37,7 +42,17 @@ function calculateMenAverageAge(people, century) {
  * @return {number}
  */
 function calculateWomenAverageAge(people, withChildren) {
-  // write code here
+  const women = people.filter(person => {
+    return withChildren !== undefined
```

Comment:
pls see [falsy values](https://developer.mozilla.org/en-US/docs/Glossary/Falsy)
```suggestion
    return withChildren
```